Source/Scene/GltfLoaderUtil.js

```diff
@@ -66,6 +66,7 @@ GltfLoaderUtil.getImageIdFromTexture = function (options) {
  * @param {Object} options Object with the following properties:
  * @param {Object} options.gltf The glTF JSON.
  * @param {Object} options.textureInfo The texture info object.
+ * @param {Object} [options.compressedTextureNoMipmap] Whether the texture is compressed and has an embedded mipmap.
```

Comment:
Some suggestions: changed `{Object}` to `{Boolean}`, added a default value (the default value should also be set in the code below with `defaultValue`), and updated the description.

```
 * @param {Boolean} [options.compressedTextureNoMipmap=false] True when the texture is compressed and does not have an embedded mipmap.
```